Module4.1_Intro_to_Data_Structures_and_Algos/activities/Act2_HashTables/7.md

```diff
@@ -1,19 +1,50 @@
-<!--title={Hash Table as a Dictionary in Python}-->
+### Linear Probing
```

Comment:
Add an intro/transitional sentence to the whole card about what they're learning in this card

Reply:
I resolved this issue.